javascript/javascript3/week3/canvasPart1_main.js

```diff
@@ -0,0 +1,35 @@
+//Lets make some art using classes
+const body = document.querySelector("body");
+const myCanvasToDrawCircle = document.getElementById("canvasId");
+const ctx = myCanvasToDrawCircle.getContext("2d");
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    ctx.beginPath();
+    ctx.arc(x, y, r, startAngle, endAngle);
+    ctx.fillStyle = fillColor;
+  }
+  draw() {
```

Comment:
It would be good all the drawing logic here, making it more clear so it is reusable and it gets executed once you call your .draw() method